app/components/specific/map/RaceMap.js

```diff
@@ -13,6 +13,7 @@ const ASPECT_RATIO = width / height;
 const LATITUDE_DELTA = 0.57;
 const LONGITUDE_DELTA = LATITUDE_DELTA * ASPECT_RATIO;
 
+// TODO: helper function to be moved asap!
 const getColor = (id = 0) => {
   // eslint-disable-next-line
   const c = (id & 0x00ffffff).toString(16).toUpperCase();
```

Comment:
consider alternatives to choose colors
using hsl colors representation to pick your color might do the trick

Reply:
roger that!